src/backend/main/py.main/caliopen_main/message/parameters/draft.py

```diff
@@ -23,6 +25,15 @@ class Draft(NewMessage):
     body_html = ""
     body = StringType()
 
+    def validate_uuid(self, user_id):
```

Comment:
No. For now backend is still lazy:  there are lot of fields that should not be modified by user but are not checked when patching… [wip]